inception-project/inception
# Feature/2403 vue recommender sidebar

**What's in the PR**
* Move recommendation related websocket code to recommendation package
* Add websocket support for recommendation evaluation messages
* Add first draft of d3 chart to draw evaluation metrics for recommenders in recommendation sidebar
* Simplify websocket message and recommender events related to websocket

**Still todo/fix**
- [ ] overlapping lines
- [x] bug: colors do not change 
- [ ] legend and/or mouseover to show datapoints
- [ ] possibility to select other metrics than accuracy
- [ ] mark points that are datapoints in step function

**How to test manually**
* Configure some recommenders
* On the annotation page open the sidebar, the chart should add datapoints/lines when new evaluations have been calculated

**Automatic testing**

**Documentation**
